astropy/time/tests/test_ut1.py

```diff
@@ -19,10 +19,10 @@
     HAS_IERS_A = True
 
 
-class TestTimeUT1():
+@pytest.mark.remote_data
```

Comment:
Here, I did the opposite but marking the whole test class as `remote_data`. This is becase its `test_utc_to_utc1` was already marked as `remote_data`, so I am guessing the other two not being marked as `remote_data` was an oversight? Or is there a reason why the other two methods in this test class need local IERS table, but not the first one?

Reply:
Here, all the tests should work with the built-in table, so I'd go for your do-not-download setting.

Reply:
Actually, the test that is already marked `remote_data` fails when using local file because extrapolation is not allowed. I'll think of something...

Reply:
OK done.